components/Toast.js

```diff
@@ -65,6 +65,15 @@ const DEFAULT_TITLES = defineMessages({
   },
 });
 
+const getVariant = variantType => {
+  switch (variantType) {
+    case 'dark':
+      return { bg: 'rgba(49, 50, 51, 0.8)', titleColor: '#ffffff', messageColor: '#C4C7CC', crossIcon: '#E8E9EB' };
+    default:
+      return { bg: '#ffffff', titleColor: '#313233', messageColor: '#76777A', crossIcon: '#C4C7CC' };
+  }
```

Comment:
@sidverma32 : This can be simplified a bit to a `if` statement I think; 

```suggestion
  if (variantType === 'dark') {
      return { bg: 'rgba(49, 50, 51, 0.8)', titleColor: '#ffffff', messageColor: '#C4C7CC', crossIcon: '#E8E9EB' };
  } else {
      return { bg: '#ffffff', titleColor: '#313233', messageColor: '#76777A', crossIcon: '#C4C7CC' };
  }
```

Reply:
> A switch statement is usually more efficient than a set of nested ifs. ... if-else better for boolean values: If-else conditional branches are great for variable conditions that result into a boolean, whereas switch statements are great for fixed data values.

Ref: [Oreilly JS Performance](https://www.oreilly.com/library/view/high-performance-javascript/9781449382308/ch04.html#:~:text=As%20it%20turns%20out%2C%20the,than%20it%20is%20for%20switch%20.)

@SudharakaP :

Actually, i choose `switch` over `if-else` because here i want to be **flexible** with the `variant type` like in future if any other variation comes (eg: primary , secondary , dark, light etc) it would be easier for other devs to add one `case` instead adding it in `else-if `. Other is **performance** and **readability** wise i selected `switch` block.


If the above two reasons are not justified , then let me know will change it to `if-else`.

Reply:
@sidverma32 : In my view, the variant is either light or dark and we don't anticipate other values. I find the if-else more readable and your linked document also says so; 

```
The argument is that if-else is easier to read when there are fewer conditions and switch is easier to read when the number of conditions is large.
```

I wouldn't care about performance as in my view we are not getting that nit picky on performance of switch vs if-else; this would only be significant if the number of cases were very large. 😄 

I'd leave the final decision on this to @Betree. 😄

Reply:
I have no strong opinion on this. The performance gain is negligible, especially since the code will go through compilers & optimizers that won't necessarily respect the original statement.

So the focus should be on readability. I'm personally ok with both approaches.

> we don't anticipate other values
The focus of this project is to make toasts more visible. If dark toasts don't do the job, the next step for design will probably be to introduce "flashy" variants (eg. a "success" one that uses a strong green, an "error" one with a flashy red). So we can anticipate more values in the future.